.github/workflows/build.yaml

```diff
@@ -358,3 +358,37 @@ jobs:
         env:
           GHR_PATH: release/
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  raw-img:
```

Comment:
can we have a Makefile target to build this locally?

Reply:
good point! 

giving also another thought - the whole script could even be a package itself, like we do for generating the squashfs directly. The pro would be that we keep all the dependencies in the tree, so the user doesn't need any special tool to generate it. But that's just brain food for a follow-up if we think it makes sense

Reply:
target for the makefile created